observelib/observe/src/main/ballerina/src/observe/natives.bal

```diff
@@ -15,7 +15,7 @@
 // under the License.
 
 final StatisticConfig[] DEFAULT_GAUGE_STATS_CONFIG = [{ timeWindow: 600000, buckets: 5,
-    percentiles: [0.33, 0.5, 0.66, 0.99] }];
+    percentiles: [0.33, 0.5, 0.66, 0.75, 0.95, 0.99, 0.999] }];
```

Comment:
How do we decide the defaults? Would 0.25 also make sense?

Reply:
It should make sense. 0.01 and 0.05 are some other commonly used percentiles.

Reply:
Yes, it would. But we decided to only expose above values as default values; However, it provides an API, so that a user can override default values if needed.